lib/improver/cli/ensemble_calibration.py

```diff
@@ -172,14 +191,33 @@ def main(argv=None):
 
     # Load cubes
```

Comment:
Is it worth considering factoring out any of this shared code, as we have basically the same stuff in both CLIs? Or is it not worth worrying about now?

Reply:
I've removed this CLI in #945. This change is also within this PR.